GERTi/GERTiClient.lua

```diff
@@ -26,23 +26,83 @@ if (component.isAvailable("tunnel")) then
 end
 
 -- addresses
-local eAddress = 0
-local iAddress = 0
+
+-- This only matters to the "GERTi" API way down below. Should be int - see startup procedure.
+-- It is updated by handler["RegisterComplete"] if the message concerns this node.
+local iAddress = nil
 local cachedAddress = {{}, {}, {}, {}, {}}
 local addressDex = 1
 -- Tables of neighbors and connections
+-- [x]{
```

Comment:
Please change to neighbors[x]{

Reply:
Will do that, along with the comment after this for connections.